validation/standard/standard-config-mainnet.toml

```diff
@@ -21,3 +21,6 @@ scalar = [684_000, 684_000]
 decimals = [6, 6]
 blob_base_fee_scalar = [0, 10_000_000]
 base_fee_scalar = [0, 10_000_000]
+
+[system_config]
+gas_limit = [1000, 60_000_000]
```

Comment:
Min L2 gas limit should be more like 8 million. It needs to be greater than the deposit gas limit (set via MAX_RESOURCE_LIMIT in the portal contract + the amount of gas allocated to the L1 info deposit)

Reply:
For reference, on OP Mainnet the minimum gas limit is 21M. We probably should use this value as the minimum until we have a good reason to lower it further.

```
$ cast call 0x229047fed2591dbec1eF1118d64F7aF3dB9EB290 "minimumGasLimit()(uint)"
21000000 [2.1e7]
```

Reply:
[21c5062](https://github.com/ethereum-optimism/superchain-registry/pull/229/commits/21c50620edf85ad520cd46533260532b92ae1567)